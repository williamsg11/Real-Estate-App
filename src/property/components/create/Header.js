import React, {PropTypes} from 'react';
import {StyleSheet, Text, View} from 'react-native';
import colors from '../../../common/colors';

export default class Header extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired,
  };

  render() {
    const {title} = this.props;
    return (
      <View style={styles.container}>
        <Text style={styles.text}>{title}</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    paddingTop: 10,
    justifyContent: 'center',
    alignItems: 'center',
    minHeight: 100,
    backgroundColor: colors.smokeGreyLight,
    // backgroundColor:'green'
  },
  text: {
    color: colors.darkGrey,
    fontWeight: '600',
    fontSize: 20,
    textAlign: 'center',
  },
});
