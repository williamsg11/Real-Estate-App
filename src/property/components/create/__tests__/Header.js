import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './../Header';

test('renders Header', () => {
  const tree = renderer.create(<Header title="the title" />).toJSON();
  expect(tree).toMatchSnapshot();
});
