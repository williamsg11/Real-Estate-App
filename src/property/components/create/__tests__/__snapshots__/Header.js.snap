// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#484848",
        "fontSize": 20,
        "fontWeight": "600",
        "padding": 30,
        "textAlign": "center",
      }
    }
  >
    the title
  </Text>
</View>
`;
